args/args.go

```diff
@@ -92,6 +96,7 @@ func (g *GeneratorArgs) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&g.OutputPackagePath, "output-package", "p", g.OutputPackagePath, "Base package path.")
 	fs.StringVarP(&g.OutputFileBaseName, "output-file-base", "O", g.OutputFileBaseName, "Base name (without .go suffix) for output files.")
 	fs.StringVarP(&g.GoHeaderFilePath, "go-header-file", "h", g.GoHeaderFilePath, "File containing boilerplate header text. The string YEAR will be replaced with the current 4-digit year.")
+	fs.BoolVar(&g.GeneratedByComment, "go-generated-by-comment", g.GeneratedByComment, `Add a "Code generated by" comment into each file.`)
```

Comment:
do we even need a flag?

Reply:
I would drop it. It's something the generator builder decides.

Reply:
if we drop the flag how will someone using this by a command line disable it?

Reply:
That seems right to me